apps/snitch_api/lib/snitch_api/accounts/accounts.ex

```diff
@@ -58,6 +65,9 @@ defmodule SnitchApi.Accounts do
     |> Account.register()
   end
 
+  @spec update_user(User.t(), map()) :: {:ok, User.t()} | {:error}
```

Comment:
When writing specs Instead of a generic `map()` it's more idiomatic to specify something like `update_params()` with `@type` 

Also, the unsuccessful return here seems to be `{:error, Ecto.Changeset.t()}`